src/showplanner/state.ts

```diff
@@ -369,12 +438,14 @@ export const removeItem = (
     movingItem.weight -= 1;
   }
 
-  // const result = await api.updateShowplan(timeslotId, ops);
-  // if (!result.every(x => x.status)) {
-  //   dispatch(showplan.actions.planSaveError("Server says no!"));
-  //   return;
-  // }
+  const result = await api.updateShowplan(timeslotId, ops);
+  if (!result.every((x) => x.status)) {
+    dispatch(showplan.actions.planSaveError("Failed to update show plan."));
+    dispatch(showplan.actions.setPlanSaving(false));
```

Comment:
Nit: dispatching two or more actions in the same place is a code smell - they should really be one action (e.g. planSaveError also sets planSaving to false). Not critical.

Reply:
Oh, it already does. Woops.